backends/include/dppl_sycl_program_interface.h

```diff
@@ -0,0 +1,114 @@
+//===---- dppl_sycl_program_interface.h - DPPL-SYCL interface --*--C++ --*--===//
+//
+//               Python Data Parallel Processing Library (PyDPPL)
+//
+// Copyright 2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//===----------------------------------------------------------------------===//
+///
+/// \file
+/// This header declares a C API to create Sycl program an interoperability
+/// program defined in OpenCL. In future, API to create interoperability
+/// kernels from other languages such as Level-0 driver API may be added here.
+///
+/// \todo Investigate what we should do when we add support for Level-0 API.
+///
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "dppl_data_types.h"
+#include "dppl_sycl_types.h"
+#include "Support/DllExport.h"
+#include "Support/ExternC.h"
+#include "Support/MemOwnershipAttrs.h"
+
+DPPL_C_EXTERN_C_BEGIN
+
+/*!
+ * @brief Create a Sycl program from an OpenCL SPIR-V binary file.
+ *
+ * Sycl 1.2 does expose any method to create a sycl::program from a SPIR-V IL
+ * file. To get around this limitation, we need to use the Sycl feature to
+ * create an interoperability kernel from an OpenCL kernel. This function first
+ * creates an OpenCL program and kernel from the SPIR-V binary and then using
+ * the Sycl-OpenCL interoperability feature creates a Sycl kernel from the
+ * OpenCL kernel.
+ *
+ * The feature to create a Sycl kernel from a SPIR-V IL binary will be available
+ * in Sycl 2.0, at which point we may deprecate this function.
```

Comment:
It is better to say, "at which point this function may become deprecated."

Reply:
I changed the comment. Thanks.